package.json

```diff
@@ -12,7 +12,7 @@
   "main": "app.js",
   "scripts": {
     "lint": "standard",
-    "start": "electron app.js --disable-http-cache",
+    "start": "browserify src/index.js > bundle.js && electron app.js --disable-http-cache",
```

Comment:
ok good to see we're not the only ones struggling with how to publish modules that work with different bundlers...